<?xml version="1.0"?>

<!-- Launch Ridgeback driver. -->
<launch>
  <!-- <remap from="/joint_states" to="/rb_joint_states" /> -->

  <rosparam command="load" file="$(find upright_ros_interface)/config/ridgeback.yaml" />
  <param name="robot_description"
    command="$(find xacro)/xacro $(find upright_ros_interface)/config/ridgeback_transmission.urdf" />

  <node name="twist_interface" pkg="upright_ros_interface" type="twist_interface" />

  <node name="controller_spawner" pkg="controller_manager" type="spawner"
        args="ridgeback_vel_traj_controller" />
</launch>
